lisa/sut_orchestrator/platform_utils.py

```diff
@@ -52,3 +53,17 @@ def get_mshv_version(node: Node) -> str:
     except Exception as e:
         node.log.debug(f"error on run mshv: {e}")
     return result
+
+
+def get_hyperv_host_version(node: Node) -> str:
+    """
+    Get Hyper-V Host Build version by calling Azure platform's _get_host_version.
+    This allows baremetal and hyperv platforms to reuse Azure's implementation
+    without duplicating code.
+    """
+    from .azure.platform_ import AzurePlatform
+
+    # Create a temporary Azure platform instance to access the method
+    # We only need the method, not the full platform initialization
+    azure_platform = AzurePlatform.__new__(AzurePlatform)
+    return azure_platform._get_host_version(node)
```

Comment:
If the method is used in other platforms, move it out from Azure platform.

Reply:
moved to platform common